NickvisionTubeConverter.GNOME/Program.cs

```diff
@@ -38,6 +38,7 @@ public Program(string[] args)
             @"* Added the option to prevent suspend while using Parabolic
               * Improved bitrate used for audio-only downloads with the best quality
               * Parabolic will now prefer videos with the h264 codec when downloading in the mp4 format. If space is a concern, users are recommended to download in the webm format which uses the vp* codec
+              * If overwriting files is off and a download's file name exists, a numbered prefix will be attached to the end of the file name to prevent error
```

Comment:
```suggestion
              * If overwriting files is off and a download's file name exists, a numbered suffix will be attached to the end of the file name to prevent error
```